### DISCUSSION TITLE
Anyway to programatically set step

### DISCUSSION THREAD

**Discussion by Twoscore23** [vueform/vueform#275] [Questions]
I am probably using this in a very niche way, but I have a form that I save the progress of on every step. So if a user leaves or refreshes the page I load the form with the existing data and ideally set the user to the step they left off on. Anyway I can achieve that?

**Comment by adamberecz** [comment]
Probably these two recipes can help you:
https://vueform.com/recipes/submit-data-on-form-steps
https://vueform.com/recipes/complete-form-steps-up-until-a-point
